File: Belajar_Day_2/Day2_5.java
import java.util.Scanner;

public class Day2_5 {

    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        String nama = "";
        int umur = 0;
        Double gaji = 0.0;

        while (true){
            try {
                CustomerHelper.inputCustomer(nama, umur, gaji);

                if (!CustomerHelper.tanyaLanjut(scan)){
                    break;
                }
            } catch (Exception e){
                System.out.println("Input error: "+ (e.getMessage() != null ? e.getMessage(): ""));
            }
        }
    }
}

File: Belajar_Day_2/Day2_5.py
from CustomerHelper import logic, input_cust, ask_continue,display_result

while True:
    try:
        nama, umur, penghasilan = input_cust()
        decision = logic(umur, penghasilan)
        display_result(nama, decision)
        if not ask_continue():
            break

    except ValueError:
        print('Input error')
    except Exception as e:
        print(e)
